android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,297 @@
+package ch.milosz.reactnative;
+
+import android.util.Log;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.WritableArray;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+import java.util.List;
+
+// import us.zoom.sdk.FreeMeetingNeedUpgradeType;
+import us.zoom.sdk.InMeetingAudioController;
+import us.zoom.sdk.InMeetingChatMessage;
+import us.zoom.sdk.InMeetingEventHandler;
+import us.zoom.sdk.InMeetingService;
+import us.zoom.sdk.InMeetingServiceListener;
+import us.zoom.sdk.InMeetingUserInfo;
+
+public class RNZoomUsInMeetingServiceListener implements InMeetingServiceListener {
+    private final static String TAG = "RNZoomUsInMeetingServiceListener";
+    private final ReactApplicationContext reactContext;
+    private final InMeetingService inMeetingService;
+
+    public RNZoomUsInMeetingServiceListener(ReactApplicationContext context, InMeetingService inMeetingService) {
+        reactContext = context;
+        this.inMeetingService = inMeetingService;
+    }
+
+    private void notifyEvent(String event, WritableMap params) {
+        WritableMap map = Arguments.createMap();
+        map.putString("event", event);
+        map.putMap("payload", params);
+        reactContext.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
+                .emit("InMeetingEvent", map);
+    }
+
+    private WritableMap extractUserInfoFromUserId(long userId) {
+        InMeetingUserInfo userInfo = inMeetingService.getUserInfoById(userId);
+        WritableMap map = Arguments.createMap();
+        map.putString("name", userInfo.getUserName());
+        map.putString("userId", "" + userId);
+        // map.putString("participantId", userInfo.getParticipantID());
+        return map;
+    }
+
+    @Override
+    public void onWebinarNeedRegister() {
+
+    }
+
+    @Override
+    public void onMeetingNeedPasswordOrDisplayName(boolean b, boolean b1, InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onJoinWebinarNeedUserNameAndEmail(InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onMeetingNeedColseOtherMeeting(InMeetingEventHandler inMeetingEventHandler) {
+
+    }
+
+    @Override
+    public void onMeetingFail(int i, int i1) {
+
+    }
+
+    @Override
+    public void onMeetingLeaveComplete(long l) {
+    }
```

Comment:
Can these be removed until they are implemented?
Or at least could you put `TODO` comment inside of each of them?